z_infra_provisioning/cloud_infra_center/ocp_upi/wait-for-bootstrap-complete.yaml

```diff
@@ -0,0 +1,48 @@
+# =================================================================
+# Copyright 2021 https://github.com/openshift/installer
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =================================================================
+
+# Required Python packages:
+#
+# ansible
+# openstackclient
+# openstacksdk
+
+- hosts: localhost
+  gather_facts: no
+
+  tasks:
+  - name: Wait-for bootstrap-complete
+    command: "./openshift-install wait-for bootstrap-complete"
+    async: 3600
+    poll: 0
+    register: bootstrap_complete_sleeper
+
+  - name: 'Waiting for bootstrap-complete'
+    async_status:
+      jid: "{{ bootstrap_complete_sleeper.ansible_job_id }}"
+    register: job_result
+    until: job_result.finished
+    retries: 1200
+    delay: 30
+
+  - name: oc get nodes
```

Comment:
Is it necessary to check node, or just want to show the result?

Reply:
I try to add the show result command, but failed. Ansible shows the message without any format, the result looks ugly.